.github/workflows/release.yml

```diff
@@ -249,11 +253,7 @@ jobs:
     - name: Download linux-x64 artifact
       uses: actions/download-artifact@v3
       with:
-        name: azureauth-${{ github.event.inputs.version }}-linux-x64.deb
-    - name: Download linux-arm64 artifact
-      uses: actions/download-artifact@v3
-      with:
-        name: azureauth-${{ github.event.inputs.version }}-linux-arm64.deb
+        name: azureauth-linux

```

Comment:
Huh, so this allows both `.deb` artifacts to be referred to by a single name (`azureauth-linux`) and downloads both of them into place?

Reply:
Right. Artifacts are similar as a zip file. When downloading an artifact, it actually downloads then extracts the inside files.